LayoutTests/http/tests/site-isolation/touch-events/touch-event-coordinates.html

```diff
@@ -0,0 +1,28 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script src="/js-test-resources/js-test.js"></script>
+<script src="/js-test-resources/ui-helper.js"></script>
+<script>
+if (window.testRunner) {
+    jsTestIsAsync = true;
+    testRunner.dumpAsText();
+}
+
+let events = [];
+addEventListener("message", (event) => {
+    events.push(event.data);
+    if (events.length == 3) {
+        if (events[0] == "touchstart 40,40" && events[1] == "touchmove 90,90" && events[2] == "touchend 90,90")
```

Comment:
shouldBe('events[0]', 'touch start 40,40');
shouldBe(...)
...

That way if the test starts failing we'll see more useful information other than just that it failed.